Integrations/IntSight/IntSight.py

```diff
@@ -441,7 +441,7 @@ def close_alert():
     alert_id = demisto.getArg('alert-id')
     reason = demisto.getArg('reason')
     free_text = demisto.getArg('free-text')
-    is_hidden = demisto.getArg('is-hidden')
+    is_hidden = demisto.getArg('is-hidden') == 'True'
```

Comment:
Why not strtobool()?

Reply:
dont really like `strtobool` in command args, since the user can pass whatever there and can cause weird issues, so prefer that as idiot-proof